packages/wonder-blocks-breadcrumbs/docs.md

```diff
@@ -0,0 +1,25 @@
+```js
+const Link = require("@khanacademy/wonder-blocks-link").default;
+const {BreadcrumbsItem} = require("@khanacademy/wonder-blocks-breadcrumbs");
+
+<Breadcrumbs>
+  <BreadcrumbsItem>
+    <Link href="https://khanacademy.org/">Home</Link>
+  </BreadcrumbsItem>
+    <BreadcrumbsItem>
```

Comment:
nit: this is indented too far

Reply:
That's a good observation, I think I'm relying to much in Prettier 😄. Do you know if there's a setting in VS Code to format code snippets inside markdown files?